client/src/store.tsx

```diff
@@ -22,6 +37,16 @@ const initialState: State = {
     production: new URL('/api/v1/', window.location.origin).href,
     test: 'https://tutv-dev.herokuapp.com/api/v1/',
   }[NODE_ENV],
+
+  // note: due to a bug, TypeScript marks code below this IIFE as "unreachable." This will be
+  // resolved in TypeScript 3.9. See https://github.com/microsoft/TypeScript/issues/36828
+  auth: (() => {
+    try {
+      return JSON.parse(localStorage.getItem('auth') || "this isn't json");
+    } catch (e) {
+      return { accessToken: null, refreshToken: null };
```

Comment:
It is possible to have more catastrophic error throwing here? Failing to login is a big deal and I think communicating that with throwing an error makes more sense.

Reply:
This doesn't handle failure to log in - this handles an inability to read a previously saved login state from `localStorage`. In other words, it only handles cases when the user isn't logged in to begin with. 

The behavior of initializing `accessToken` and `refreshToken` as null in this case just means that the user is logged out, and will be presented with the `SignIn` page.

Reply:
Gotcha. I like `null`, `null` is great. Bigly